Core/Registry/RegistryManager.cs

```diff
@@ -314,14 +314,14 @@ private void LoadOrCreate()
                 log.ErrorFormat("Uncaught exception loading registry: {0}", ex.ToString());
                 throw;
             }
-
             AscertainDefaultRepo();
         }
 
         private void Create()
         {
             log.InfoFormat("Creating new CKAN registry at {0}", path);
             registry = Registry.Empty();
+            AscertainDefaultRepo();
```

Comment:
Was there a case where the the repositories didn't get populated? It's also run at the end of `loadOrCreate()`, which is the only place we call `Create()`.

Reply:
I think I wanted to _move_ it into `Create` since that would make more sense and so that we would not be calling `Save()` with missing repo info. Then I had to re-add it back to `LoadOrCreate` due to some wrinkle.

Reply:
Okay. Guess there's no harm in keeping it twice.